### ISSUE TITLE
enable overriding favicon from Helmet component

### ISSUE THREAD

**Issue by giuliaghisini** [plone/volto#6013]
if i want to load favicon differently than loading it from /public folder, i have to override the /helpers/Html.js component because favicon is statically linked here and i'm not able to override it by Helmet component.

**Comment by stevepiercy** [comment]
Is this a duplicate of either?

- https://github.com/plone/volto/issues/4654
- https://github.com/plone/volto/issues/3617

**Comment by giuliaghisini** [comment]
> Is this a duplicate of either?
> 
> * [How to put public folder in an addon? #4654](https://github.com/plone/volto/issues/4654)
> * [Allow addons to extend the contents of the `public` directory with their own `public` folder #3617](https://github.com/plone/volto/issues/3617)

no.. here we need to set favicon from an api response, not from a static folder. Otherwise, i think that for all this 3 issues the solution is the same.. maybe the solutions are two:

1. move the markup of favicon inside a specific component that will be overridable
2. make Helmet component able to set/render the favicon markup
